test/system/tag_test.rb

```diff
@@ -51,5 +51,10 @@ def setup
     end
 
   end
-
+  
+  test 'adding a location tag to a user profile' do
+    visit "/profile/jeff"
```

Comment:
Hmm. So, is it possible the `jeff` user has an unexpected state? I have two debugging ideas -- cc @Manasa2850 @coder645 @gauravahlawat81 

1. Manually run `$('.blurred-location-input').click()` on the JavaScript console while on a profile page in your browser to see what happens
2. we could search for/assert the existence of other things before trying to run this code, to confirm that we are in the state we expect. For example, something like:

```ruby
assert !users(:jeff).has_power_tag('lat') && !users(:jeff).has_power_tag('lon')
```

code ref: 

https://github.com/publiclab/plots2/blob/2b2bd90290f22d5a5ff709b25b1a40e11094db7e/app/models/user.rb#L171

Reply:
> Manually run `$('.blurred-location-input').click()` on the JavaScript console while on a profile page in your browser to see what happens

Please refer to this video -> [Video](https://drive.google.com/file/d/1mHlux37OTWuBRPFhzbxhhUQwiXJ6RVW9/view?usp=sharing)

>we could search for/assert the existence of other things before trying to run this code, to confirm that we are in the state we expect.

Seems like a good idea to me !

Reply:
Hi @Manasa2850 - can you confirm you're doing exactly the steps in the video @gauravahlawat81 has posted? And if not, what exactly was different? Then we can craft or adjust the test!

Reply:
Hi @jywarren yes I've followed the same steps. I've attached a gif depicting the before-after times too (in my first comment)
Thanks!